gensim/models/word2vec.py

```diff
@@ -413,6 +414,21 @@ def __init__(
         self.null_word = null_word
         self.train_count = 0
         self.total_train_time = 0
+
+        RULE_DEFAULT = 0
+        RULE_DISCARD = 1
+        RULE_KEEP = 2
+
+        if trim_rule:
+            def keep_vocab_item(word, count, min_count_):
```

Comment:
same here; the `self.trim_rule` attribute is not pickleable if it contains such anonymous functions or lambdas.

Also, why the `_` at the end of `min_count_`?